.env.example

```diff
@@ -1,9 +1,9 @@
 VITE_API_URL=https://acc-jungle-gym-api.herokuapp.com
 VITE_SESSION_DB_URL=localhost
-VITE_SESSION_DB_USERNAME=false
+VITE_SESSION_DB_USERNAME=
```

Comment:
Dit was toch al in ander MR. Nja, oke

Reply:
Ja ging fout.. oeps. moet ik dat weghalen?

Reply:
Nee hoor. Is pr ima toch zo